.github/workflows/release.yml

```diff
@@ -0,0 +1,92 @@
+name: Create GitHub Release
+
+env:
+  RUNTIME_DEPENDENCIES: '[ libudev1, libevdev2, libglib2.0-0, libgee-0.8-2, upower ]'
+  UBUNTU_RELEASE: jammy
+  UBUNTU_PUBKEY: 871920D1991BC93C
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        platform:
+          - linux/amd64
+          - linux/arm64
+    steps:
+      - uses: actions/checkout@v4
+      - uses: docker/setup-qemu-action@v2
+      - uses: docker/setup-buildx-action@v2
+      - uses: docker/build-push-action@v4
+        with:
+          platforms: ${{ matrix.platform }}
+          outputs: build
+          build-args: |
+            UBUNTU_RELEASE=${{ env.UBUNTU_RELEASE }}
+      - name: Prepare AppImage build
+        env:
+          TARGET_PLATFORM: ${{ matrix.platform }}
+        shell: bash
+        run: |
+          set -eu
+
+          mkdir -p AppDir
+          find build -mindepth 1 -maxdepth 1 -type d | while read -r path; do
+            cp -a "${path}" AppDir/usr
+            build_arch="$(basename "${path}")"
+
+            icon_path=AppDir/usr/share/icons/hicolor/scalable/mimetypes
+            icon_name=no-icon
+            mkdir -p "${icon_path}"
+            touch "${icon_path}/${icon_name}.svg"
+
+            touch dummy-to-prevent-fail-of-appimage-builder.AppImage.zsync
+
+            set -a
+            APPIMAGE_ARCH="${build_arch}"
+            APPDIR_APPINFO_ID="$(echo "${GITHUB_REPOSITORY}" | tr / .)"
```

Comment:
Wouldn't this end up set to `v1993.evdevhook2`? That's incorrect, the right value would be [`org.v1993.evdevhook2`.](https://github.com/v1993/evdevhook2/blob/082a06d2dacb71236153002b0c2ff592e7c1c704/src/main.vala#L33C21-L33C41)

Reply:
Also - most of these are static values and thus should be set directly in config file IMO.

Reply:
I simplified the templating. The static values are now managed in the `AppImageBuilder.yml`.